Makefile

```diff
@@ -1,7 +1,7 @@
 all: build
 
-GIT_COMMIT:=$(shell git rev-list -1 HEAD)
-GIT_LAST_TAG:=$(shell git describe --abbrev=0 --tags)
+GIT_COMMIT:=$(shell git rev-parse --short HEAD)
```

Comment:
can you explain those changes?

Reply:
Oh, that change was unnecessary. I was thinking of returning a short commit hash to not to format it in the root